### PR TITLE
Update for 3.2.6.SP1-redhat-00001

### PR DESCRIPTION
**Pull request by llowinge**

### CODE REVIEW

**Comment by llowinge** [comment]
@ldrozdo Can you please review (mainly the docs). I'll tag Viliam later.

**Comment by vkasala** [comment]
Adding comment

**Comment by llowinge** [comment]
@vkasala Thanks. I'll merge after @ldrozdo approval.

**Comment by ldrozdo** [comment]
@llowinge LGTM, I just had some question about minor details but otherwise it can be merged.

File: README.md [hunk 1]
```diff
@@ -36,24 +36,86 @@ After a new version of Quarkus is released, you should do the following:
...
+In such case the theoretical steps should be:
+
+* Creating PR with new tag added to https://github.com/openshift/release/blob/master/core-services/image-mirroring/supplemental-ci-images/mapping_supplemental_ci_images_ci#L129
+* After merged, creating another PR with changes to `.yaml` files (probably with help of INTEROP team)
+
 
 ## How to run locally
-- Change `oc_login.sh` to login to your existing OCP cluster.
+### Without OCP deployment
+**_NOTE:_** This is easier approach, but it is not 100% accurate to what happens on OpenShift CI. But usually such verification is enough.
```
[the remark this answers is not in the document]
Comment (ldrozdo) [reply README.md:110]: @llowinge do you see the review now?

File: README.md [hunk 2]
```diff
@@ -36,24 +36,86 @@ After a new version of Quarkus is released, you should do the following:
 - Update this section of Dockerfile (description is in the comments):
 ```asciidoc
 ENV CAMEL_QUARKUS_BRANCH=2.13.x # branch in of the repo which is used to run tests
-ENV QUARKUS_VERSION=2.13.8.SP1-redhat-00003 # version of Quarkus BOM associated with the release
+ENV QUARKUS_VERSION=2.13.8.SP1-redhat-00003 # version of Quarkus BOM (usually platform one) associated with the release
```
[the remark this answers is not in the document]
Comment (llowinge) [reply README.md]: I've rewritten it.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (llowinge) [reply README.md:110]: Yes, i've rewritten it.

**Review by ldrozdo — APPROVED** [review]

**Comment by ldrozdo** [comment]
@llowinge looks great, thanks

**Pull request merged**